client-spark/common/src/main/java/org/apache/spark/shuffle/RssShuffleUtils.java

```diff
@@ -175,4 +179,44 @@ public static ShuffleManager loadShuffleManager(String name, SparkConf conf, boo
     CoordinatorClientFactory coordinatorClientFactory = new CoordinatorClientFactory(clientType);
     return coordinatorClientFactory.createCoordinatorClient(coordinators);
   }
+
+  public static void applyDynamicClientConf(SparkConf sparkConf, Map<String, String> confItems) {
+    if (sparkConf == null) {
+      LOG.warn("Spark conf is null");
+      return;
+    }
+
+    if (confItems == null || confItems.isEmpty()) {
+      LOG.warn("Empty conf items");
+      return;
+    }
+
+    for (Map.Entry<String, String> kv : confItems.entrySet()) {
+      String confKey = kv.getKey();
+      String confVal = kv.getValue();
+      if (!sparkConf.contains(confKey) || RssClientConfig.RSS_MANDATORY_CLUSTER_CONF.contains(confKey)) {
+        LOG.warn("Use conf dynamic conf {} = {}", confKey, confVal);
+        sparkConf.set(confKey, confVal);
+      }
+    }
+  }
+
+  public static final Set<StorageType> STORAGE_TYPE_NEED_NO_PATH =
```

Comment:
We should make a method public not a constant.

Reply:
I may not quite understand your point, add a method to hold the storage type constants?